dotcom-rendering/src/components/ExpandableMarketingCard.tsx

```diff
@@ -156,16 +163,21 @@ const sectionStyles = css`
 	gap: ${space[3]}px;
 	border-top: 1px solid ${neutral[100]};
 	padding-top: ${space[2]}px;
+`;
 
-	h3 {
-		${headlineBold17};
-	}
+const subheadingStyles = css`
+	${headlineBold17};
+`;
 
-	p {
-		${textSans15}
-		margin-right: ${space[4]}px;
-		z-index: 1;
-	}
+const paragraphStyles = css`
+	${textSans15}
+	margin-right: ${space[4]}px;
+	z-index: 1;
```

Comment:
Minor - could this use [getZindex()](https://github.com/guardian/dotcom-rendering/blob/094336dc0898e64712ccc19de83c16deb15d1e15/dotcom-rendering/src/lib/getZIndex.ts) ?

Reply:
It's a bit tricky. The component has a z-index of 8 as that is suitable for the other elements on the page. This z-index is just so the text appear on top of the illustrations in the component. Given this is all within the component, I think it would add noise to `getZIndex` which controls the order of things at a page level.